### ISSUE TITLE
About Padding Problem

### ISSUE THREAD

**Issue by LinanYaooo** [qjadud1994/CRNN-Keras#30]
Hi, thanks for your code above all.
And I am using the repo as an initial stage of a wild filed OCR task for unfixing length sequence.

How can I adjust the code to avoid the "invalid path problem of CTC"? I've searched some resources saying that there is a constraint between input length and label length but I was still confused the meaning.

Since my training set's maximum length string is over 35 words, and I just wonder if my padding will influence the performance of the task and how to really achieve the variant length sequence work properly, thanks
